### ISSUE TITLE
[Reranker效率问题疑惑] 发现Reranker的一个效率问题

### ISSUE THREAD

**Issue by Tian14267** [FlagOpen/FlagEmbedding#988]
各位大神好。
我在测试reranker的效率时，发现一个问题。如下：

**Reranker源码：`./FlagEmbedding/flag_reranker.py`
测试设备：GPU（A800）
测试数据量：1000条
测试数据的token长度：2048
测试batch：4**

描述：针对 `self.tokenizer` 和 `self.model `进行了时间监控，如图：
![000](https://github.com/user-attachments/assets/fcddf7e9-0044-44d2-bb19-2a93178c0c6b)

测试出来的相关结果如图：
![1](https://github.com/user-attachments/assets/2a4dce1f-0f89-46f3-888c-a3cbd3394457)
从这个结果来看，仿佛是 文本转token耗时较长（再细致拆分发现，是token从CPU转GPU耗时较长）

但是！当我把`scores = self.model(**inputs, return_dict=True).logits.view(-1, ).float()` 注释之后，在统计效率，理论上总耗时应该就是减去模型的耗时（4秒）。
![2](https://github.com/user-attachments/assets/9ca074b0-b363-442b-8629-3b6ad194cdff)

但是我发现，跑出来的耗时却是：
![3](https://github.com/user-attachments/assets/817a942c-2dda-4ab3-bd4d-7f7f9a99f85e)
 token转换竟然基本不占时间。

请问下这个是什么情况啊？
